src/Model/Profile.php

```diff
@@ -947,6 +947,16 @@ public static function getTabs($a, $is_owner = false, $nickname = null)
 			];
 		}
 
+		if (x($_SESSION, 'new_member') && $is_owner) {
```

Comment:
Can you please replace `x($_SESSION, 'new_member')` by `!empty($_SESSION['new_member'])` ? We're currently trying to get rid of `x()` and any help is appreciated.